core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/DefaultTokenFactoryRegistry.java

```diff
@@ -35,13 +35,13 @@ public DefaultTokenFactoryRegistry(InternalDriverContext context) {
   public TokenFactory tokenFactoryFor(String partitioner) {
     if (partitioner.endsWith("Murmur3Partitioner")) {
```

Comment:
Is it really realistic to imagine that a different Murmur3 partitioner could exist other than `org.apache.cassandra.dht.Murmur3Partitioner`? And even so, shouldn't we error out in that case instead of assume that it's the same partitioner? I wonder if we shouldn't simply store the FQCN of each partitioner as a public constant field in each `TokenFactory` implementation and then reference them here using `equals` instead of `endsWith`.